src/helpers/discord_utils.ts

```diff
@@ -101,6 +101,7 @@ export async function sendErrorMessage(message: Eris.Message<Eris.GuildTextableC
 export async function sendOptionsMessage(message: Eris.Message<Eris.GuildTextableChannel>, guildPreference: GuildPreference, updatedOption: string) {
     let totalSongs = await getSongCount(guildPreference);
     let groupsMode = guildPreference.getGroupIds() !== null;
+    let goalMode = guildPreference.getGoal() !== 0;
```

Comment:
Guild preference should have a boolean method that returns whether a goal is set or not